### DISCUSSION TITLE
Is it possible to open the compiled tex file in another folder in zathura?

### DISCUSSION THREAD

**Discussion by frisbro303** [helix-editor/helix#7352] [Q&A]
I'm trying to create a workflow for using latex with helix and zathura. I've created a solution that works well, based on things written at the github project. The only issue is, that it generates a lot of messy files. Using the `--output-directory=tmp` option, i'm able to output the compiled latex file to a temporary directory. My language config is shown below. 
Is it possible to open the generated pdf file in the directory in zathura?
Thanks for the awesome text editor. 

best regards

`
[[language]]
name = "latex"
language-servers = ["texlab", "ltex-ls"  ]

[language-server.ltex-ls]
command ="ltex-ls"

[language-server.texlab.config.texlab.build]
onSave = true
forwardSearchAfter = true
args = ["--output-directory=tmp"]

[language-server.texlab.config.texlab.forwardSearch]
executable = "zathura"
args = ["--synctex-forward", "%l:1:%f", "%p"]

[language-server.texlab.config.texlab.chktex]
onEdit = true
`

**Comment by JurajLieskovsky** [comment]
Try adding
```
[language-server.texlab.config.texlab]
auxDirectory = "tmp"
```
to your config.
